api/handlers/portal_link.go

```diff
@@ -247,7 +248,8 @@ func (h *Handler) LoadPortalLinksPaged(w http.ResponseWriter, r *http.Request) {
 	}
 
 	endpointIDs := getEndpointIDs(r)
-	filter := &datastore.FilterBy{EndpointIDs: endpointIDs}
+	ownerId := r.URL.Query().Get("ownerId")
```

Comment:
@jirevwe In fact, this is all we need to solve the bug. 

```go
var portalLinkEndpoints []string 
endpointRepo := postgres.NewEndpointRepo(h.A.DB, h.A.Cache)
endpoints, err := endpointRepo.FindEndpointsByOwnerID(r.Context(), pl.ProjectID, ownerId)
if err != nil {
   return nil, err
}

for _, endpoint := range endpoints {
   portalLinkEndpoints = append(endpointIDs, endpoint.UID)
}
```

This feels weird, but that means you can retrieve portal links with both ownerId and endpointIds at the same time except we want to restrict that behavior.

Reply:
@jirevwe If we do this, we can solve the bug temporarily without the other changes.